src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/ContentHelper.Common.cs

```diff
@@ -87,6 +87,7 @@ private static bool CheckIsText(string contentType)
             || CheckIsXml(contentType)
             || CheckIsJson(contentType);
 
+            #if !CORECLR // Registry access not supported on CoreCLR
```

Comment:
PowerShell Core on Windows uses the registry. Use Platform.IsWindows to make it a runtime check.